### ISSUE TITLE
git diff fatal exception

### ISSUE THREAD

**Issue by vk3il** [xlwings/git-xl#65]
git-xl has been working well for some time, but I've recently had it failing on my main excel project file. The error is as follows:

```
$ git diff
[21228] Failed to execute script diff
Traceback (most recent call last):
  File "diff.py", line 47, in <module>
  File "diff.py", line 12, in get_vba
  File "site-packages\oletools\olevba3.py", line 2793, in extract_all_macros
  File "site-packages\oletools\olevba3.py", line 2738, in extract_macros
  File "site-packages\oletools\olevba3.py", line 2749, in extract_macros
  File "site-packages\oletools\olevba3.py", line 1324, in _extract_vba
  File "site-packages\oletools\olevba3.py", line 1301, in check_value
oletools.olevba3.UnexpectedDataError: Unexpected value in VBA/dir for variable PROJECTLCID_Id: expected 0002 but found 004A!
fatal: external diff died, stopping at CDR compliance tool.xlsb
```

I tried creating a new excel workbook with a simple VBA macro and it too throws the same error. The environment is Microsoft® Excel® for Microsoft 365 MSO (16.0.14131.20278) 64-bit 

Seems that the PROJECTLCID_Id value is wrong in the file, but Excel doesn't seem to mind and running a repair doesn't fix the problem. Has MS changed this value perhaps?

**Comment by fzumstein** [comment]
Hi @vk3il, thanks for reporting! this looks like an issue with https://github.com/decalage2/oletools. I'll try to find the time to upgrade to the latest version to see if that solves the issue.

**Comment by vk3il** [comment]
Thanks @fzumstein . Looking at that repository, it may in fact be a newly opened bug - see: [https://github.com/decalage2/oletools/issues/701](https://github.com/decalage2/oletools/issues/701)
Maybe let them know of the downstream impact of it?

**Comment by alzec** [comment]
Hello,

Does anyone know if there is a temporary solution for this problem?

**Comment by caiocesart** [comment]
> Hello,
> 
> Does anyone know if there is a temporary solution for this problem?

Hi, have you found any solution?

**Comment by dnsnx** [comment]
Same here. Would be interested in a solution to this issue.

**Comment by junxu94** [comment]
Hi, I am seeing the exact same issue, but not to see solution in this thread. Can someone provide the solution? Thanks so much!

**Comment by jspraul** [comment]
> Update to newest version of oletools, update for python 3.10.4 #70

I think you have to clone and build the latest from this repository, as nothing has been released yet.

**Comment by fzumstein** [comment]
Yeah, to make a new release, I'd need to port the pipeline over to GitHub actions, and that takes a lot of time a nerves (that I currently a bit short of).

**Comment by winnieXY** [comment]
Any news on that? I'm hitting the same issue

**Comment by fzumstein** [comment]
this is now released with 0.5.1
